spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkWriteConf.java

```diff
@@ -198,7 +198,7 @@ private DistributionMode adjustWriteDistributionMode(DistributionMode mode) {
     }
   }
 
-  public DistributionMode copyOnWriteDeleteDistributionMode() {
```

Comment:
Given the recent changes, I am no longer sure we need separate methods for copy-on-write and merge-on-read.